### DISCUSSION TITLE
Item Metadata

### DISCUSSION THREAD

**Discussion by emroch** [danricho/where_the#11] [Ideas]
The ability to store item metadata would turn this from a simple organizational tool into a full inventory/asset tracking tool.
Examples of useful metadata:

- Item value
- Item quantity
- Origin (manufacturer, store, supplier, etc)
- Date of purchase
- original purchase price
- long-form description

The type of metadata required for each user is likely different, so these should be generic dictionary items, not a fixed set. However, it would be useful to define a list of possible metadata tags globally and have a drop-down in each item to select and set the metadata so you don't end up with typos that screw up display/search/sort. If a user decides some new item warrants a new metadata entry, they could add it to the list and it would become available for all items.

**Comment by danricho** [comment]
Considering all of what you've said, I'd be interested to know if [Inventree](https://inventree.readthedocs.io/en/latest/) would be of use?
